tests/blackbox.rs

```diff
@@ -0,0 +1,283 @@
+//! Tests lesspass.rs using lesspass-cli as oracle.
+//!
+//! The test vectors in this file are generated by lesspass-cli using
+//! the following script.
+//!
+//! ```sh
```

Comment:
Do you mind putting the script in its own file, and instead linking to that file in the comment?

Additionally, just for a "prettier" `cargo test` output, I made two small changes below.

Reply:
Sure thing.